neo/core/analogsignal.py

```diff
@@ -623,3 +625,33 @@ def merge(self, other):
         if hasattr(self, "lazy_shape"):
             signal.lazy_shape = merged_lazy_shape
         return signal
+
+    def splice(self, signal, copy=False):
```

Comment:
AnalogSignal.time_slice() is get a slice from.
AnalogSignal.slice() is put a slice into.
Maybe we could have something more consistent.
No ?

Reply:
splice, not slice

Reply:
Oups, sorry.
I was too tired to read it correctly.